README.md

```diff
@@ -2,28 +2,133 @@
 
 Website: [https://codexappaa.herokuapp.com/](https://codexappaa.herokuapp.com/)
 
+## What is CodeX?
+
+CodeX is an online publishing platform, for members of the tech industry, inspired by [Medium.com](https://medium.com/). On it, you can write, read, and connect with others, on such topics as:
+- software engineering
+- computer science
+- data science
+- machine learning
+- and more.
+
+### Did You Know?
+
+The codex, a bound collection of handwritten contents on non-paper materials, is the historical ancestor of the modern book. Conveniently, for our purposes, its pronunciation and spelling imply e**x**change of information, and echo computer **code**, prominent features of innovation and technology.
+
 ## Technologies Used
 
 - JavaScript
 - Express
-- PUG
+- Pug
 - CSS
+- BCrypt
+- Heroku
+- Sequelize
+- PostgreSQL
 
 ## Features
 
-- Website home page with stories
-- User registration / login (with a demo account available)
-  - An authenticated home page with recommended stories and stories of users you follow
-- Specific story pages to view the story written by an author
-  - A comments section, where authenticated users can add comments to a story
-  - Like/unlike feature on the story and its comments that update live
-  - Follow/unfollow feature that updates live
-- A story creation page allowing authenciated users to post new stories
-  - Supports markdown in the body of the story
+### Home Page with Stories
+
+![Home Page](https://github.com/kvh8899/week13-project/blob/readme/public/images/screens-readme/homepage.png)
```

Comment:
This is going to be relevant to all image links:
When this gets merged and this branch is deleted, these paths will be broken.

When you move the images into `/docs/images` update the urls to start with 
`https://github.com/kvh8899/week13-project/blob/main/docs/images/` instead of `https://github.com/kvh8899/week13-project/blob/readme/public/images`

Reply:
Screens' directory moved to /docs/images. Links in README.md updated.